src/Database/Adapter/Postgres.php

```diff
@@ -147,6 +147,35 @@ public function createCollection(string $name, array $attributes = [], array $in
         return true;
     }
 
+     /**
+     * Get Collection Size
+     * @param string $name
+     * @return int
+     * @throws Exception
+     * 
+     */
+
+     public function getCollectionSize(string $name): int
+     {
+         $database = $this->getDefaultDatabase();
+         $name = $this->filter($name);
+     
+         $query = $this->getPDO()->prepare("
+         SELECT pg_size_pretty(pg_total_relation_size(
```

Comment:
Please double-check with pg_size_pretty() as we want data in bytes?

Reply:
I removed getCollectionSize for Postgres as it works but is giving erros in the tests.